README.md

```diff
@@ -62,25 +62,31 @@ To get a local copy up and running follow these simple example steps.
 In order to run this project you need:
 
 - Install ruby on your computer or run in with docker
-- Run `ruby main`
+- Install raiks on your computer or run in with docker
 
 ### Setup
 
 Clone this repository by using the command line :
 
-- `git clone https://github.com/maaloum/BlogApp.git`
+- `git clone https://github.com/maaloum/Blog_Rails.git`
 
 ### Install
 
 Install this project with:
 
-`npm gemfile`
+`bundle`
 
 ### Usage
 
 To run the project, execute the following command:
 
```

Comment:
- [ ] [OPTIONAL]
Kindly endeavor to remove the guide note below the MIT license, on _line 146_.